resource-watch/resource-watch
# improve embed legends

## Overview
Legend improvements on coral and embed pages,
Map embed page now has a max height on the legend so it does not overflow the map on the topics pages

Also made the z-index of the Legend a higher priority than the map interaction buttons, so the "expand & close" legend arrow is always on-top of it. So even if for some reason it overflows those buttons, it will be closable. 

## Testing instructions
For the coral page bug, you can view the embedded code here `http://localhost:9000/embed/map/bc0ceb8f-f06c-4c3c-8433-548e900ff93b`

The other one, check any topic page

## Pivotal task
https://www.pivotaltracker.com/story/show/156428089
https://www.pivotaltracker.com/story/show/156430901
